tests/unit/test_models.py

```diff
@@ -1,22 +1,19 @@
 import frontmatter
-import pytest
 
 from archivy.extensions import get_max_id
 
 attributes = ["type", "title", "desc", "tags", "path", "id"]
 
-def test_new_note(note_fixture):
+
+def test_new_note(test_app, note_fixture):
     """
     Check that a new note is correctly saved into the filesystem
     with the right attributes and the right id.
     """
-    max_id = get_max_id()
-    assert note_fixture.id + 1 == max_id
+    with test_app.app_context():
+        max_id = get_max_id()
+        assert note_fixture.id + 1 == max_id
```

Comment:
Could you remove the +1 here. Yesterday I changed get_max_id so it would return the actual max_id instead of max_id + 1. I unfortunately forgot to remove a line of code that persisted this design and I have now removed it in https://github.com/Uzay-G/archivy/commit/5a33475ce9c8aeb2a25491dd98fd255032590111.